contrib/eraser/training/config/eraser.yaml

```diff
@@ -13,12 +13,12 @@ bridge_noise_sigma: 0.05 # see Appendix A.1
 conditioning_images_keys: []
 conditioning_masks_keys: []
 image_size: [536, 960] # (H, W), while original RORD dimension is [H=540, W=960]
```

Comment:
For the removalbench images with a square ratio, we're going to loose a significant part of the content, right?... I guess it's too restrictive to work with a single aspect ratio (i.e. `[536, 960]` for now). WDYT?

Reply:
I think if we train at `[536, 960]` it's fair to make the validation at `[536, 960]`.
We do not want to measure multiple-aspect-ratio generalization (not now).

While implementing multiple-aspect-ratio we will change this.